react/finger-exercises/redux/02 - react-redux/src/app/screens/App/index.js

```diff
@@ -1,7 +1,10 @@
 import React, { Component, Fragment } from 'react';
+import PropTypes from 'prop-types';
```

Comment:
I suggest importing the property types that you use in the component, for example, in this file only use arrayOf 
```
import { arrayOf } from 'prop-types';
```
This makes the code more readable.